CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing
+
+The development flow includes:
+
+- [create workspace, clone sources](#create-workspace-clone-sources)
+- [build](#build)
+- [run tests](#run-tests)
+- [package binaries](#package-binaries)
+- [code](#code)
+- [push changes, provide pull request](#push-changes-provide-pull-request)
+
+## Create workspace, clone sources
+
+To create a workpace you can use [devfile](devfile.yaml).
+For that you have to [install the latest version](https://www.eclipse.org/che/docs/che-7/che-quick-starts.html#installing-the-chectl-management-tool) of this chectl and create a workspace:
```

Comment:
it looks odd to install chectl when we clone chectl ?

Reply:
Yes and no. It's ok if we are explaining the user steps to create a workspace.
From other side it's extra information as user is already pretty experienced if he is going to contribute to chectl.

Reply:
Extra sentence has been removed.